src/__init__.py

```diff
@@ -355,8 +357,10 @@ def adjust_filler_items(self, item_pool, traps):
                 extra_item = self.create_item(self.random.choice(traps))
                 item_pool.append(extra_item)
 
+            filler_name_to_add = self.get_filler_item_name()
```

Comment:
Can I get you to inline this? If people are using the hook, there's a reasonable chance it wants to return a different item each time it makes one, so we really want to request a name once per create_item.

Reply:
That was the intent of creating the hook

Reply:
There are three potential ways that people might want to update `filler_item_name`:

1. Updating `filler_item_name` before gen / needing nothing from world.
2. Updating `filler_item_name` to a single choice during gen and accounting for options, etc. from world.
3. Updating `filler_item_name` on each loop of Manual's filler padding during adjust_filler_items, also accounting for options, etc. from world.

1 is separate from `get_filler_item_name()` (and has been supported for a long time anyways). 2 and 3 are both potential use cases for `get_filler_item_name()` in relation to Manual's padding of filler items. So I updated the PR to support 2 and 3 both by making `filler_item_name` a prop on world that people can update via other hooks (for 2).